src/shared-components/container/index.js

```diff
@@ -1,18 +1,3 @@
-import PropTypes from 'prop-types';
-
-import {
-  Container,
-  Section,
-  Divider,
-  Image,
-} from './style';
-
-Container.Section = Section;
```

Comment:
This was causing issues because `Rollup` was turning this into

    import {Container, Section Divider, Image } from 'src/shared-components/container/styles';
    import 'src/shared-components/container';

Webpack was tree shaking the `import 'src/shared-components/container'` since we marked the package as side effect free.  Basically, any mutations like this need to happen before exporting.  I went through the rollup code to see if I could find any other potential issues and I couldn't find anything.

Reply:
Yep, this looks like the only instance. Good catch!